docs/contributing-coding-guidelines.md

```diff
@@ -21,13 +21,13 @@ your editor.
 
 ### Minimum Supported `rustc` Version
 
-Wasmtime supports the current stable version of Rust.
+Wasmtime supports the latest three stable releases of Rust.
 
 Cranelift supports stable Rust, and follows the [Rust Update Policy for
```

Comment:
While we're here, could we update this for Cranelift as well? We're no longer included in the mozilla-central tree and aren't subject to their Rust version policies anymore; it's probably uncontroversial to say now that Cranelift will follow the same policy as Wasmtime?